src/pages/home/report/ContextMenu/PopoverReportActionContextMenu.js

```diff
@@ -260,15 +260,19 @@ class PopoverReportActionContextMenu extends React.Component {
 
     confirmDeleteAndHideModal() {
         this.callbackWhenDeleteModalHide = () => this.onComfirmDeleteModal = this.runAndResetCallback(this.onComfirmDeleteModal);
-        Report.deleteReportComment(this.state.reportID, this.state.reportAction);
+
+        if (this.state.reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.IOU) {
+            // @TODO: Implement new DeleteMoneyRequest API command from issue https://github.com/Expensify/Expensify/issues/270502
```

Comment:
I suppose this is a valid reason to leave a TODO

Reply:
Yea, but I'm not sure that we should merge this before the method is implemented. This affects IOUs so it'd be confusing for the user if they tried to delete an IOU and nothing happened